docs/platform/reference/service-ip-address.rst

```diff
@@ -0,0 +1,14 @@
+Service IP address and DNS
+==========================
+
+When a new service is created in Aiven, a cloud service provider will dynamically assign a public IP address from their connection pool. This IP address is not a permanent one, because with every service node update (failover, maintenance upgrade or cloud migration) the IP address will change as Aiven will create a new VM, migrate to it and then retire the older node. 
```

Comment:
You can rrefer here about static IPs as well https://docs.aiven.io/docs/platform/concepts/static-ips.html and how to obtain them https://docs.aiven.io/docs/platform/howto/static-ip-addresses.html